### ISSUE TITLE
Allow PublicApiAnalyzers to work with single PublicApi.txt file

### ISSUE THREAD

**Issue by ashmind** [dotnet/roslyn-analyzers#3281]
#### Analyzer package

[Microsoft.CodeAnalysis.PublicApiAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.PublicApiAnalyzers)

#### Package Version

v2.9.8 (Latest)

#### Diagnostic ID

[RS0016]

#### Suggestion

This analyzer seem useful for any libraries, including small ones. However in my small libraries there is no much difference between Shipped/Unshipped -- it's either there, or not.

I suggest that a single `PublicAPI.txt` file should also enable the analyzer -- in that case analyzer will only read that file, and will add to that file.

**Comment by gtbuchanan** [comment]
Yes, it would be nice if this were an option for folks with faster release cycles and/or who don't want an extra release step. This is the main thing holding me back from switching from PublicApiGenerator.

**Comment by sorensenmatias** [comment]
For our release cycle, I would really love this as well. Any news?
